network.js

```diff
@@ -30,5 +30,24 @@ export default {
       ],
     },
   },
+  icon: '~/assets/images/currencies/atom.png',
+  governanceLinks: [
```

Comment:
should we include this or just delete it?

Reply:
I was also thinking delete

Reply:
Removed